io.github.mpobaschnig.Vaults.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "io.github.mpobaschnig.Vaults",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "vaults",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--filesystem=host"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
+        "test-args" : [
+            "--socket=x11",
+            "--share=network"
```

Comment:
same as above. no x11, no network.

Reply:
I think anything related to `test` arguments is being ignored by flatpak-builder in the sandbox mode.

Reply:
Removed.